week-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,61 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-a.jpg",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-d.jpg",
+  "./assets/cute-cat-e.jpg",
+  "./assets/cute-cat-f.jpg",
 ];
 
+const forwardSlide = document.querySelector("#forward-btn");
+const backwardSlide = document.querySelector("#backward-btn");
+const imageSelector = document.querySelector("#carousel-img");
+const stopBtn = document.querySelector("#stop-btn");
+const autoBackBtn = document.querySelector("#auto-back-btn");
+const autoForwardBtn = document.querySelector("#auto-forward-btn");
 
-// Write your code here
\ No newline at end of file
+let num = 0;
+imageSelector.style.width = "600px";
+imageSelector.style.height = "400px"; // Adjusted height for better proportions
+
+forwardSlide.addEventListener("click", function () {
+num++;
+if (num === images.length) {
+  num = 0;
```

Comment:
Im not quite sure what num is supposed to be. For the future, tru to name variables something people can understand just by looking at your code!